entrypoint/push.js

```diff
@@ -219,6 +219,8 @@ export const send = RavenLambdaWrapper.handler(Raven, async (event) => {
             };
         }
 
+        await sleep(1000);
```

Comment:
Before, or after `new Telegram`? Should be fine there, huh?

Reply:
doesn't matter, but here is nicer